doc/orchestrator.rst

```diff
@@ -165,3 +165,287 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multidb
+=======
+
+To support computationally intensive tasks like scientific simulations,
+SmartSim provides functionality to automate the deployment of multiple
+databases on an HPC cluster. As data volume increases, consider augmenting
+the number of sharded databases or provision multiple standalone databases,
+to ensure the workload can scale to meet resource demands. To ensure that data
+retrieval, updates, and synchronization are carried out smoothly and efficiently
```

Comment:
Also not quite accurate, it's more of a necessity that when you have more than one database, you need to some way to distinguish them